### PR TITLE
Remove defaults.yaml

### PR DESCRIPTION
**Pull request by kalantar**
Addresses issues https://github.com/iter8-tools/etc3/issues/138 and https://github.com/iter8-tools/etc3/issues/164

### CODE REVIEW

**Comment by kalantar** [comment]
Also fixed a bug in `controllers/weights.go`

File: api/v2alpha2/defaults.go [hunk 1]
```diff
@@ -19,14 +19,27 @@ package v2alpha2
 
 import (
 	"time"
-
-	"github.com/iter8-tools/etc3/configuration"
 )
 
 const (
 	// NoneHandler is the keyword users can use to indicate no handler
```
Comment (sriumcp) [inline api/v2alpha2/defaults.go:25]: Why do we have this string "None" when handler itself is gone away? In any case, the comment doesn't seem right (users no longer indicate none handler through keywords).

**Review by sriumcp — CHANGES_REQUESTED** [review]
None handler seems to require changes (removal?)

File: api/v2alpha2/defaults.go (hunk 1 above)
Comment (kalantar) [reply api/v2alpha2/defaults.go]: Good catch. Fixed.

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**